src/poetry/utils/password_manager.py

```diff
@@ -172,12 +172,29 @@ def set_pypi_token(self, name: str, token: str) -> None:
         else:
             self.keyring.set_password(name, "__token__", token)
 
-    def get_pypi_token(self, name: str) -> str | None:
-        if not self.keyring.is_available():
-            token: str | None = self._config.get(f"pypi-token.{name}")
-            return token
-
-        return self.keyring.get_password(name, "__token__")
+    def get_pypi_token(self, repo_name: str) -> str | None:
+        """Get PyPi token.
+
+        First checks the environment variables for a token,
+        then the configured username/password and the
+        available keyring.
+
+
+        Parameters
```

Comment:
While there are not a lot of docstring usages in the code base describing arguments, return values and exceptions, the few of them that do use [Sphinx](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format) format, like here for instance:
https://github.com/python-poetry/poetry/blob/820c503fb788c514838c757fee175f71b9eb3212/src/poetry/repositories/installed_repository.py#L48-L50

It would be preferable to follow the same convention IMO, if we start adding more docstrings.

Reply:
Sure, I just didn't find any and kinda forgot others exist because I use NumPy almost always

Reply:
No worries, there **really** are not a lot of usage of it, so it was not easy to know :smile: 
```console
$ git grep ":param" | wc -l
22
```

Reply:
Okay, I just updated it, should be good 😁